langwatch/src/server/api/routers/publicEnv.ts

```diff
@@ -21,6 +21,8 @@ export const publicEnvRouter = publicProcedure
       IS_ATLA_DEFAULT_JUDGE: shouldUseAtlaModelForJudges(),
       POSTHOG_KEY: env.POSTHOG_KEY,
       POSTHOG_HOST: env.POSTHOG_HOST,
+      HAS_LANGWATCH_NLP_SERVICE: !!env.LANGWATCH_NLP_SERVICE,
```

Comment:
I just realized in production we don't have this variable either, it should really be `!!env.LANGWATCH_NLP_SERVICE || !!env.LANGWATCH_NLP_LAMBDA_CONFIG`